Assets/Scripts/Models/Time/WorldTime.cs

```diff
@@ -294,13 +315,13 @@ public int Year
 
     #region IFormattable implementation
 
-    string IFormattable.ToString(string format, IFormatProvider provider = null)
+    string IFormattable.ToString(string format, IFormatProvider provider)
```

Comment:
Yes, if you'd be so kind as to drop in the non provider one. Just have it set to null, so that the picking the right culture thing will work right (well, will work if/when we rename the language files).

Also, in case you didn't notice, you have a Stylecop failure.